catkin_virtualenv/src/catkin_virtualenv/venv.py

```diff
@@ -26,6 +26,11 @@
 import shutil
 import subprocess
 import tempfile
+try:
+    from urllib.request import urlretrieve
+except:
```

Comment:
Needs an ImportError to pass lint I believe

Reply:
Thank you! I fixed it.